utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    private const PACKAGES_WITHOUT_MONOREPO_AND_PROJECT_BASE = [
+        'http-smoke-testing',
+        'coding-standards',
+        'read-model',
+        'product-feed-google',
+        'product-feed-zbozi',
+        'product-feed-heureka-delivery',
+        'product-feed-heureka',
+        'plugin-interface',
+        'framework',
+        'form-types-bundle',
+        'migrations',
+        'google-cloud-bundle',
+    ];
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and push git tags for packages excluding monorepo and project-base';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 640;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $versionString = $version->getVersionString();
+
+        $this->processRunner->run('mkdir temp');
+        foreach (self::PACKAGES_WITHOUT_MONOREPO_AND_PROJECT_BASE as $packageName) {
+            $this->processRunner->run('cd temp && git clone https://github.com/shopsys/' . $packageName . '.git');
+            $this->processRunner->run('cd temp/' . $packageName . ' && git checkout ' . $this->initialBranchName . ' && git tag ' . $versionString);
+            $this->processRunner->run('cd temp/' . $packageName . ' && git push origin ' . $versionString);
```

Comment:
Can there be some way of ensuring that it will tag the right commits? Remember that tagging a package is something **we cannot revert**. Maybe instead of pushing the tags right away, I would create an array of package names and commit msgs (package name is `sprintf('shopsys/%s', $packageName)` and you can get the commit msg by executing `'git show ' . $versionString . ' --format="%s"'`).

After the developer confirms that correct commits are tagged, the releaser could loop through all repositories in `temp/`, push the tags and remove `temp/`. This would seem a lot safer to me.

Reply:
Not all packages will actually have the same commit. There is nothing changed in coding-standards and monorepo-tools, as they have no dependencies on `shopsys/*` packages...